config/locales/views.youth.de.yml

```diff
@@ -8,6 +8,10 @@ de:
     fields_youth:
       nationality:
         placeholder: z.B. CH
+      nationality_j_s:
+        CH: CH
+        FL: FL
+        ANDERE: ANDERE
```

Comment:
Braucht es nicht, wenn immer die Übersetzung aus den Models genommen wird. Gilt für alles in den `views.youth.*.yml`-Dateien